src/twisted/python/test/test_url.py

```diff
@@ -178,7 +184,7 @@ def test_repr(self):
             repr(URL(scheme=u'http', host=u'foo', path=[u'bar'],
                      query=[(u'baz', None), (u'k', u'v')],
                      fragment=u'frob')),
-            "URL.fromText(%s)" % (repr(u"http://foo/bar?baz&k=v#frob"),)
+            "URL.from_text(%s)" % (repr(u"http://foo/bar?baz&k=v#frob"),)
```

Comment:
This is just the `__repr__` - [`URL.fromText`](https://github.com/mahmoud/hyperlink/blob/4f56ee8b7fcc3e6ca4ba13dbb8b55368cfca0433/hyperlink/_url.py#L1098-L1100 still exists).